crates/table/src/table_index/mod.rs

```diff
@@ -280,11 +293,15 @@ impl TypedIndex {
                 AlgebraicType::U16 => Self::UniqueDirectU16(<_>::default()),
                 AlgebraicType::U32 => Self::UniqueDirectU32(<_>::default()),
                 AlgebraicType::U64 => Self::UniqueDirectU64(<_>::default()),
+                // For a plain enum, use ~`u8` as the native type.
```

Comment:
```suggestion
                // For a plain enum, use `u8` as the native type.
```

Was this tilde intentional?